editor/src/core/shared/import/proejct-health-check/requirements/requirement-storyboard.ts

```diff
@@ -0,0 +1,157 @@
+import type { ProjectContentTreeRoot } from 'utopia-shared/src/types'
+import type { ProjectRequirements, RequirementCheckResult } from '../utopia-requirements-types'
+import { RequirementResolutionResult } from '../utopia-requirements-types'
+import type { RequirementCheck } from '../utopia-requirements-types'
+import { defaultEither } from '../../../../../core/shared/either'
+import { StoryboardFilePath } from '../../../../../components/editor/store/editor-state'
+import { getPackageJsonFromProjectContents } from '../../../../../components/editor/store/editor-state'
+import {
+  addFileToProjectContents,
+  getProjectFileByFilePath,
+} from '../../../../../components/assets'
+import { codeFile } from '../../../../../core/shared/project-file-types'
+import { addStoryboardFileToProject } from '../../../../../core/model/storyboard-utils'
+
+export default class CheckStoryboard implements RequirementCheck {
+  getRequirementName(): keyof ProjectRequirements {
+    return 'storyboard'
+  }
+  getStartText(): string {
+    return 'Checking for storyboard.js'
+  }
+  check(projectContents: ProjectContentTreeRoot): RequirementCheckResult {
+    return createStoryboardFileIfNecessaryInner(projectContents)
+  }
+}
+
+export function createStoryboardFileIfNecessary(
+  projectContents: ProjectContentTreeRoot,
+): ProjectContentTreeRoot {
+  const result = createStoryboardFileIfNecessaryInner(projectContents)
+  return result.newProjectContents ?? projectContents
+}
+
+function createStoryboardFileIfNecessaryInner(
```

Comment:
from here to the end of the file it is just code that was moved from `actions.tsx` (no logic change)